lms/session.py

```diff
@@ -1,8 +1,6 @@
 """The app's Pyramid session."""
 from pyramid.session import JSONSerializer, SignedCookieSessionFactory
 
-__all__ = []
```

Comment:
Mypy wanted a type annotations here (it doesn't know which values will be in the list).

Just deleted it.